apps/api/v2/src/modules/auth/guards/organizations/is-admin-api-enabled.guard.ts

```diff
@@ -54,7 +54,7 @@ export class IsAdminAPIEnabledGuard implements CanActivate {
       }
     }
 
-    const org = await this.organizationsRepository.findById(Number(organizationId));
+    const org = await this.organizationsRepository.findById({ id: Number(organizationId) });
```

Comment:
Now that we are extending features/OrganizationRepository, findById already exists in it and it accepts an object with id